src/potions.ts

```diff
@@ -189,6 +189,10 @@ class Potion {
   }
 }
 
+/**
+ * Determines if potions are worth using by comparing against meat-equilibrium. Considers using pillkeeper to double them. Accounts for non-wanderer embezzlers. Does not account for PYEC/LTC, or running out of turns with the ascend flag.
+ * @param doEmbezzlers Do we account for embezzlers when deciding what potions are profitable?
```

Comment:
` * @param {boolean} [doEmbezzlers=false] Do we account for embezzlers when deciding what potions are profitable?`